### ISSUE TITLE
Check text overlap 

### ISSUE THREAD

**Issue by Valexandre** [cj-holmes/paintr#2]
Would there be a way to prevent text overlap for the numbers on an image ?
<img width="422" alt="imagerdn" src="https://user-images.githubusercontent.com/12694649/119839368-68906500-bf04-11eb-9fbf-2da9571e4221.png">
I feel like there could be less text density for lines.

**Comment by cj-holmes** [comment]
Yeah this is more diffcult. Every individual area to colour will have it's own number...even if the area is a single pixel size in the original image. 
So around edges you sometimes get lots of small areas to colour (like below) The numbers are not along the lines.
![image](https://user-images.githubusercontent.com/25186045/119948090-6542bb00-bf90-11eb-9dad-3de41c3ff638.png)

You can try smoothing the image more, or using a different colour palette?

Of course, a better fix would be to not create the very small areas in the first place. I tried to do this but couldn't get it to work very well. If you can think of a way of doing it i'd love to know about it!

If you save the image at the required size, are the numbers still overlapping?
